packages/core/src/components/VerticalNavigation/VerticalNavigation.test.tsx

```diff
@@ -208,6 +208,23 @@ const SliderSample = () => {
 };
 
 describe("VerticalNavigation", () => {
+  const { ResizeObserver } = window;
+
+  beforeEach(() => {
+    // @ts-ignore
+    delete window.ResizeObserver;
+    window.ResizeObserver = vi.fn().mockImplementation(() => ({
+      observe: vi.fn(),
+      unobserve: vi.fn(),
+      disconnect: vi.fn(),
+    }));
+  });
+
+  afterEach(() => {
+    window.ResizeObserver = ResizeObserver;
+    vi.restoreAllMocks();
+  });
```

Comment:
I needed this in my PR as well.
I think we should move to the global test setup, as this is part of the DOM APIs that JSDOM currently doesn't mock

Reply:
I opened a PR for this https://github.com/lumada-design/hv-uikit-react/pull/3354

Reply:
Nice, I'll leave this as is then.